examples/text_classification/iterable_train.py

```diff
@@ -15,8 +15,32 @@
 
 from tqdm import tqdm
 
+r"""
+This example shows how to build an iterable dataset from the iterator. A
+new API get_csv_iterator() is used to read CSV file for the data. An abstract
```

Comment:
I suggest we write "The function get_csv_iterator" instead of "A new API", since API typically refers to sets of functions.